### DISCUSSION TITLE
Snakemake and Jupyter Notebooks

### DISCUSSION THREAD

**Discussion by s-canchi** [nih-cfde/training-and-engagement#427] [Snakemake]
Can you describe when workflows like Snakemake are better to use than Jupyter notebooks?

**Comment by marisalim** [comment]
Jupyter notebooks are great for interactive analysis, but may be too slow for longer processes that take more than a few minutes to complete. In those cases, you may prefer to run your analyses with snakemake workflows on a high performance computer cluster or on the cloud (i.e., AWS, Google). The outputs of those workflows can be further explored interactively and shared with Jupyter notebooks!
